src/client/gui/lib/update_available.dart

```diff
@@ -15,14 +17,39 @@ class UpdateNotifier extends Notifier<UpdateInfo> {
 
   void set(UpdateInfo updateInfo) {
     if (updateInfo.version.isBlank) return;
+
     final updateNotificationExists = ref.read(notificationsProvider).any((n) {
       return n is UpdateAvailableNotification && n.updateInfo == updateInfo;
     });
     if (updateNotificationExists) return;
-    ref
-        .read(notificationsProvider.notifier)
-        .add(UpdateAvailableNotification(updateInfo));
+
+    // Existing in-app notification
+    ref.read(notificationsProvider.notifier).add(UpdateAvailableNotification(updateInfo));
```

Comment:
Was this re-formatted automatically? If I run `dart format` on the file, it reverts it to the way it was before.
If your IDE automatically formats code, make sure it uses `dart format` instead of its own rules.